netbox/netbox/choices.py

```diff
@@ -84,7 +84,7 @@ class ButtonColorChoices(ChoiceSet):
     """
     Map standard button color choices to Bootstrap 3 button classes
     """
-    DEFAULT = 'outline-dark'
+    DEFAULT = 'outline-secondary'
```

Comment:
IMO we should default to an empty value or the string "default," and avoid hard-coding a specific color/style name (so we don't end up in this situation again down the road).

Reply:
changed to default, the link isn't rendered in the template so had to add some code to netbox/extras/templatetags/custom_links.py to change from default to outline-secondary.